src/user/user.controller.ts

```diff
@@ -8,6 +8,7 @@ export class UserController {
   constructor(private readonly userService: UserService) {}
 
   @Post()
+  @UsePipes(new ValidationPipe())
   create(@Body() createUserDto: CreateUserDto) {
     return this.userService.create(createUserDto);
```

Comment:
Aq vc tem q alterar o status code pra 201, se não passar o status code, ele retorna o 200.
Corrigindo, ficaria assim:
return res.status(HttpStatus.CREATED).json(this.userService.create(createUserDto));